app/src/main/res/layout-land/fragment_columns.xml

```diff
@@ -81,39 +80,28 @@
                 app:layout_constraintStart_toStartOf="parent"
                 app:layout_constraintTop_toBottomOf="@+id/button_add" />
 
-            <Switch
-                android:id="@+id/switch_invert"
-                android:layout_width="wrap_content"
-                android:layout_height="wrap_content"
-                android:layout_marginBottom="4dp"
-                android:text=""
-                android:theme="@style/AppTheme.Switch.Invert"
-                app:layout_constraintBottom_toBottomOf="parent"
-                app:layout_constraintEnd_toEndOf="parent"
-                app:layout_constraintStart_toStartOf="parent" />
-
             <Button
                 android:id="@+id/button_remove"
                 android:layout_width="35dp"
                 android:layout_height="35dp"
-                android:layout_marginBottom="4dp"
                 android:background="@null"
                 android:drawableStart="@drawable/ic_remove"
                 app:layout_constraintBottom_toTopOf="@+id/switch_invert"
                 app:layout_constraintEnd_toEndOf="parent"
+                app:layout_constraintHorizontal_bias="0.51"
```

Comment:
Why do we need it?

Reply:
I don't know, it was in previous version in several places, i decided not to touch it.
I may remove it, if necessary.